t/ui/12-needle-edit.t

```diff
@@ -518,6 +518,16 @@ subtest 'Showing new needles limited to the 5 most recent ones' => sub {
       or diag explain \@needle_names;
 };
 
+my %tests
+  = (empty => '', string => '"workaround"', hash => '{ "name": workaround", "value": "workaround for bsc#123456" }');
```

Comment:
for a line break like this I suggest to put some parameters into a local variable, e.g.:

```perl
my $hash = '{ "name": workaround", "value": "workaround for bsc#123456" }';
my %tests = (empty => '', string => '"workaround"', hash => $hash);
```